docs/FAQ.md

```diff
@@ -223,6 +223,114 @@ Questions about VictoriaMetrics can be asked via the following channels:
+
+Yes, you can replace your entire Prometheus stack with VictoriaMetrics
+
+
+### I have two Prometheus replicas in my k8s cluster that write data into VictoriaMetrics and therefore see dual data from node_exporter’s in my Grafana dashboard. I have tried to use  `-dedup.minScrapeInterval: 10s`  parameter for vmagent https://docs.victoriametrics.com/#deduplication but my Pod was disconnected because it didn’t know this flag.
+
+`-dedup.minScrapeInterval` supported by single node version and for cluster version it supported by vmselect/vmstorage services so you need to set this flag for them.
+
+
+### How can I build VictoriaMetrics on FreeBSD?
+
+To build VictoriaMetrics on FreeBSD you need to install Golang in OS and then run gmake victoria-metrics-pure or install it from FreeBSD Ports - see [this link](https://www.freebsd.org/cgi/ports.cgi?query=victoria&stype=all) .
+
+
+### What is the rule of thumb for vminsert replication factor? What if I have 8 vmstorage pods?
+
+E.g. replication factor 3 means that you have your initial data copied 3 times and these copies are spread across 8 nodes. Every node will therefore have 3/8 of the initial data.
+
+
+### Can I make a query to different tenants at the same time? 
+
+No. You can only do this if you will run the query through [promxy](https://github.com/jacksontj/promxy).
+
+
+### How does vmselect know in which storage to find the entire dataset?  
+
+`vmselect` queries all the `vmstorage` instances. See [these docs](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#replication-and-data-safety) for details.
+
+
+### Is there a way to use vm to identify the top 10 slow queries or non-performing queries being made to VictoriaMetrics?
+
+There is an `/api/v1/status/top_queries`  which should help to identify slowest queries.
+
+
+### How can I set replication files between vmstorages after one node has died? vminsert send data to all nodes, but after I reboot one node has less data than the node that was working. Is there any other way to manage this other than with rsync?
+
+There is currently no rebalancing between nodes. Please see the following links for more details. 
+https://github.com/VictoriaMetrics/VictoriaMetrics/issues/188
+https://github.com/VictoriaMetrics/VictoriaMetrics/issues/991
+
+
+### Does VictoriaMetrics support the many-to-many metrics multiplication?
+
+Many-to-many is not supported yet. There are only one-to-one and one-to-many - see [this] (https://prometheus.io/docs/prometheus/latest/querying/operators/#vector-matching)
+
+
+### Is it a good idea to write into Single Node VictoriaMetrics from different protocols like Graphite plaintext protocol, Influxdb and Prometheus metrics? Maybe it would be better to keep these data separate from each other?
+
+This is a good case. Please ensure to have different label sets for the written data to avoid collisions
```

Comment:
I think, sentence about collisions is not needed. Or if you think it is, then it should be explained better. Otherwise, I have an impression that I always need to add an extra label if I use more than one protocol